kernel-kit/build.sh

```diff
@@ -197,7 +197,7 @@ FW_URL=${FW_URL:-http://distro.ibiblio.org/puppylinux/firmware}
 # $package_name_suffix $custom_suffix $kernel_ver
 kernel_version_full=${kernel_version}${custom_suffix}
 kernel_srcsfs_version=${kernel_version}
-aufs_utils_git="git://git.code.sf.net/p/aufs/aufs-util.git"
+aufs_utils_git="https://git.code.sf.net/p/aufs/aufs-util"
```

Comment:
Why? https works but the original works fine here too (and is the dedicated one for git).
 What is the error that you get  with ``` git clone git://git.code.sf.net/p/aufs/aufs-util.git```?